mlrun/runtimes/databricks_runtime_wrapper.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+def run_mlrun_databricks_job(
+    context,
+    internal_code,
+    token_key="DATABRICKS_TOKEN",
+    timeout=20,
+    **kwargs,
+):
+    import datetime
+    import json
+    import uuid
+
+    from databricks.sdk import WorkspaceClient
+    from databricks.sdk.service.compute import ClusterSpec
+    from databricks.sdk.service.jobs import Run, SparkPythonTask, SubmitTask
+
+    import mlrun
+
+    logger = context.logger
+
+    def upload_file(workspace: WorkspaceClient, script_path_on_dbfs: str):
+        modified_code = internal_code
+        with workspace.dbfs.open(script_path_on_dbfs, write=True, overwrite=True) as f:
+            f.write(modified_code.encode("UTF8"))
+
+    workspace = WorkspaceClient(token=mlrun.get_secret_or_env(key=token_key))
+
+    now = datetime.datetime.now()
+    formatted_date_time = now.strftime("%Y-%m-%d_%H-%M-%S")
+    script_path_on_dbfs = (
+        f"/home/{workspace.current_user.me().user_name}/mlrun_databricks_runtime/"
+        f"sample_{formatted_date_time}_{uuid.uuid4()}.py"
```

Comment:
Any reason that this should be called "sample"? Seems like an artifact from test/proof-of-concept code.